ci/ray_ci/pipeline/gap_filling_scheduler.py

```diff
@@ -60,9 +61,33 @@ def get_gap_commits(self) -> List[str]:
             .split("\n")
         )
 
-    def _trigger_build(self, commit: str) -> int:
-        # TODO(can): Implement this method
-        pass
+    def _find_blocked_build_and_job(
+        self, commit: str
+    ) -> Tuple[Optional[Dict], Optional[Dict]]:
```

Comment:
could you add a comment on what the return values are? or maybe create a simple class/TypedDict for the return value?

why is the build number a dict? and job id a dict too?